packages/admin-client/src/common/contexts/CatchmentContext/CatchmentContext.tsx

```diff
@@ -0,0 +1,94 @@
+import { createContext, useState, useEffect, useMemo } from "react"
+
+const createCatchmentName = (catchmentNo: number, location: string) => {
+    let catchmentName = catchmentNo < 10 ? "0" : ""
+    return catchmentName + catchmentNo.toString() + " - " + location
+}
+
+const getCatchments = () => {
+    // eslint-disable-next-line prefer-const
+    let result: any[] | undefined = []
```

Comment:
result will never be undefined if you're instantiating it as an empty array and only doing pushes on it